tests/modules/programs/zellij/enable-shells.nix

```diff
@@ -3,7 +3,12 @@
 {
   programs = {
     # All shell integratons are enabled by default.
```

Comment:
```suggestion
```

Reply:
Thanks. It is still early in the morning where I'm at XD